io/EptReader.cpp

```diff
@@ -940,6 +983,8 @@ void EptReader::load()
 
             if (m_info->dataType() == EptInfo::DataType::Laszip)
                 readLaszip(nodeBuffer->view, key, nodeId);
+            else if (m_info->dataType() == EptInfo::DataType::Zstandard)
+                readZstandard(nodeBuffer->view, key, nodeId);
```

Comment:
Default here is different from above.  Seems better to be explicit for all cases.

Reply:
96a672f33f4b550e13ffb3bc1acf3b4c7f1c4f8c